public_suffix_list.dat

```diff
@@ -12158,6 +12158,12 @@ translated.page
 // Submitted by Jacob Lee <jacob@autocode.com>
 autocode.dev
 
+// Autonomous Database OCI - Oracle Dyn https://cloud.oracle.com/home https://dyn.com/dns/
```

Comment:
Should be added here
 https://github.com/publicsuffix/list/pull/2109/files#diff-b4fc013fa0dff30019626bef00fba7a39210a002b463194b258b8b8ae302215bL12527

Reply:
Resolved the order recommendation and moved after Customer-OCI